libs/SalesforceRestAPI/SalesforceRestAPI.xcodeproj/project.pbxproj

```diff
@@ -258,6 +292,16 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		CEA8838F1C19135D008D871B /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				CEA883A61C191461008D871B /* libSalesforceSDKCore.a in Frameworks */,
```

Comment:
That's correct.  Leave the target dependencies (so the dependent targets will build first, deploy their headers, etc.), but no "linking" phase for the library targets.